exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -0,0 +1,145 @@
+# Unison Basics
+
+## Terms and functions
+
+Terms and functions are the basic units of writing a Unison program.
+
+A "term" is what we call a static value. It's defined with `name = expression`, optionally preceded by a type signature with the form `name : MyType`. Unison supports type inference, so you might see the type signature omitted for simple definitions.
+
+```
+myTerm = 42
+
+helloThere : Text
+helloThere = "Hello, World!"
+```
+
+Unison functions look like `myFunctionName argument1 argument2 = myImplementation`; they start with the name of the function followed by space-separated arguments.
+
+```
+getMax a b = Nat.max a b
+```
+
+Functions can optionally include a type signature. The last type on the right is always the final return type of the function, and each argument is separated by the function arrow `->`.
+
+```
+{{getMax accepts two arguments of type Nat and its return type is a Nat}}
```

Comment:
Up until this point, the documentation literal syntax is not explained. You could consider omitting it here and only have it later on, when the syntax is explained.